conf.d/wmi_check.yaml.example

```diff
@@ -17,6 +17,10 @@ instances:
   #
   # `namespace` is the optionnal WMI namespace to connect to (default to `cimv2`).
   #
+  # `provider` is the optional WMI provider (default to `32` on Datadog Agent 32-bit or `64`).
+  # Usefull to request WMI data from the non-default provider. Available options are: `32` or `64`.
+  # For more information: https://msdn.microsoft.com/en-us/library/aa393067(v=vs.85).aspx
```

Comment:
So after discussing this: by default let's keep the default provider that WMI chooses. This way we're sure not to break anything for the existing users of the WMI checks.

Consequence of this: let's keep the documentation on the `provider` option